src/locales/es/texts.json

```diff
@@ -0,0 +1,287 @@
+    "addCollections": "Presione enter para añadir una colección",
+    "addFilter": "Añadir filtro",
+    "addItemIds": "Presione enter para añadir el ID de un elemento",
+    "additionalFilters": "Filtros adicionales",
+    "buttons": {
+      "filter": "Filtro",
+      "reset": "Restablecer"
+    },
+    "dateDescription": "Todas las horas en Coordinated Universal Time (UTC).",
+    "equalTo": "igual a",
+    "filterBySpatialExtent": "Filtra por extensión espacial",
+    "greaterThan": "mayor que",
+    "itemIds": "IDs de Elementos",
+    "itemsPerPage": "Elementos por página",
+    "itemsPerPageDescription": "Número de elementos solicitados por página, máx. {maxItems} elementos.",
+    "lessThan": "menor que",
+    "likeOperatorDescription": "Puede usar carácteres comodín. `_` representa un solo carácter, `%` representa cualquier número de carácteres. Para buscar por un carácter comodín en específico, necesita añadir `\\` antes del carácter.",
+    "matches": "coincidencia",
+    "modifyCriteria": "Por favor modifique los criterios de búsqueda.",
+    "noItemsFound": "No se encontró ningún elemento para los filtros seleccionados.",
+    "notEqualTo": "no igual a",
+    "notFullySupported": "Puede que no todas las opciones estén soportadas.",
+    "notSupported": "La API no soporta la Búsqueda por Elementos.",
+    "placeholder": "Búsqueda...",
+    "selectCollections": "Seleccione una o varias colecciones...",
+    "selectItemIds": "Seleccione una o varias IDs de Elemento...",
+    "sortOptions": {
+      "datetime": "Fecha y hora",
+      "id": "ID",
+      "title": "Título"
+    },
+    "spatialExtent": "Extensión espacial",
+    "temporalExtent": "Extensión temporal",
+    "title": "Buscar"
+  },
+  "showMore": "Mostrar más...",
+  "sidebar": {
+    "switchCatalog": "Cambiar catálogo"
+  },
+  "sort": {
+    "asc": {
+      "label": "Ascendente",
+      "title": "Ordenar en orden ascendente (A-Z)"
+    },
+    "desc": {
+      "label": "Descendente",
+      "title": "Ordenar en orden descendente (Z-A)"
+    },
+    "title": "Ordenar"
+  },
+  "source": {
+    "detailsAboutSource": "Detalles sobre la fuente STAC",
+    "label": "Fuente",
+    "language": {
+      "label": "Idioma: {currentLanguage}",
+      "onlyData": "¡Los datos sólo están disponibles en este idioma!",
+      "onlyUI": "¡El interfaz de usuario sólo está disponible en este idioma!",
+      "switch": "Cambiar el idioma"
+    },
+    "locatedAt": "El archivo de metadatos STAC se encuentra en:",
+    "share": {
+      "email": "E-mail",
+      "label": "Compartir",
+      "message": "{title} está disponible en {url}",
+      "sharePageWithOthers": "Comparta la URL de está página donde guste:",
+      "title": "Compartir",
+      "twitter": "Twitter",
+      "withOthers": "Comparta esta página con otros"
+    },
+    "stacVersion": "Versión STAC:",
+    "title": "Fuente de datos",
+    "valid": "Valida:",
+    "validating": "Validando...",
+    "validationNA": "n/a"
```

Comment:
Is this a missing translation? A dictionary showed me "no interesa" ;-)

Reply:
In this case I'd leave it like that, "n/a", because the abbreviation can be translated to "no aplica" which literally means "does not apply".